casual/casual-caller-http-client/src/main/java/se/laz/casual/http/HttpClient.java

```diff
@@ -19,13 +19,14 @@
 import java.io.Closeable;
 import java.net.URI;
 import java.util.Objects;
-import java.util.logging.Logger;
+
+import static java.lang.System.Logger.Level.*;
```

Comment:
We prefer not to use wildcard imports.

Please change to a specific import rather than wildcard `*`. 

`import static java.lang.System.Logger.Level.DEBUG;` in this case.

Update accordingly in all files please.